templates/travel/elements/_btn.css

```diff
@@ -0,0 +1,37 @@
+/**
+ * basscss btn
+ *
+ * Extend the ampstart button class.
+ */
+
+ .btn {
```

Comment:
reuse ampstart-btn and override where needed, do not create classes like this. Please follow rules for the class names (ask neal for the guideline doc.).